forge-gui/res/cardsfolder/upcoming/wyll_blade_of_frontiers.txt

```diff
@@ -0,0 +1,11 @@
+Name:Wyll, Blade of Frontiers
+ManaCost:1 R
+Types:Legendary Creature Human Warlock
+PT:1/1
+S:Mode$ Continuous | Affected$ You | AddKeyword$ If you would roll one or more dice, instead roll that many dice plus one and ignore the lowest roll. | Description$ If you would roll one or more dice, instead roll that many dice plus one and ignore the lowest roll.
+T:Mode$ RolledDieOnce | TriggerZones$ Battlefield | ValidPlayer$ You | Execute$ TrigPut | TriggerDescription$ Whenever you roll one or more dice, put a +1/+1 counter on CARDNAME.
+SVar:TrigPut:DB$ PutCounter | Defined$ Self | ValidCard$ Card.Self | CounterType$ P1P1 | CounterNum$ 1
```

Comment:
PutCounter doesn't use `ValidCard$` param
Really this doesn't need ` | Defined$ Self` or ` | CounterNum$ 1` either as those are defaults

Reply:
Is the original author still available for the tweak? I believe this MR is otherwise ready for merge, should we merge as is and tweak this in the followup MR?